src/ComputeSharp.Graphics/Buffers/ConstantBuffer{T}.cs

```diff
@@ -80,36 +69,23 @@ public override void GetData(Span<T> span, int offset, int count)
         }
 
         /// <inheritdoc/>
-        public override void SetData(ReadOnlySpan<T> span, int offset, int count)
+        /// <inheritdoc/>
```

Comment:
You duplicated the XML comment here by accident 😉

```suggestion
```

Reply:
Fixed in https://github.com/Sergio0694/ComputeSharp/pull/68/commits/c613d92413a8e45a044032e9759351ba5b305fa3.